workflow/meta/rose-meta.conf

```diff
@@ -69,6 +69,15 @@ type=character
 description=Password for Space-Track.org
 type=character
 
+#--------------------------------------------------
+# Julia
+#--------------------------------------------------
+
+[template variables=JULIA_DEBUG]
+description=JULIA_DEBUG environment variable. 
+    Sets which Julia packages have `@debug` statements active.
+    e.g. `JULIA_DEBUG="Main,IceFloeTracker"` would activate `@debug` messages in the main thread and in `IceFloeTracker.jl`
+
```

Comment:
@cpaniaguam it's documented here. I don't expect it to be something that regular users would want to do, but if you search for "debug" in the project directory, you'll find it.